src/libs/actions/Welcome/OnboardingFlowTasks.ts

```diff
@@ -0,0 +1,221 @@
+    };
+    const inviteTeamTask: OnboardingTask = {
+        type: 'inviteTeam',
+        autoCompleted: false,
+        title: translateLocal('onboarding.tasks.inviteTeamTask.title'),
+        description: ({workspaceMembersLink}) => translateLocal('onboarding.tasks.inviteTeamTask.description', {workspaceMembersLink}),
+    };
+
+    const addAccountingIntegrationTask: OnboardingTask = {
+        type: 'addAccountingIntegration',
+        autoCompleted: false,
+        title: ({integrationName}) => translateLocal('onboarding.tasks.addAccountingIntegrationTask.title', {integrationName}),
+        description: ({integrationName, workspaceAccountingLink}) => translateLocal('onboarding.tasks.addAccountingIntegrationTask.description', {integrationName, workspaceAccountingLink}),
+    };
+    const inviteAccountantTask: OnboardingTask = {
+        type: 'inviteAccountant',
+        autoCompleted: false,
+        title: translateLocal('onboarding.tasks.inviteAccountantTask.title'),
+        description: ({workspaceMembersLink}) => translateLocal('onboarding.tasks.inviteAccountantTask.description', {workspaceMembersLink}),
+    };
+
+    const startChatTask: OnboardingTask = {
+        type: 'startChat',
+        autoCompleted: false,
+        title: translateLocal('onboarding.tasks.startChatTask.title'),
+        description: translateLocal('onboarding.tasks.startChatTask.description'),
+    };
+    const splitExpenseTask: OnboardingTask = {
+        type: 'splitExpense',
+        autoCompleted: false,
+        title: translateLocal('onboarding.tasks.splitExpenseTask.title'),
+        description: translateLocal('onboarding.tasks.splitExpenseTask.description'),
+    };
+
+    const reviewWorkspaceSettingsTask: OnboardingTask = {
+        type: 'reviewWorkspaceSettings',
+        autoCompleted: false,
+        title: translateLocal('onboarding.tasks.reviewWorkspaceSettingsTask.title'),
+        description: translateLocal('onboarding.tasks.reviewWorkspaceSettingsTask.description'),
+    };
+
+    const onboardingEmployerOrSubmitMessage: OnboardingMessage = {
+        message: translateLocal('onboarding.messages.employerOrSubmitMessage'),
+        tasks: [selfGuidedTourTask, submitExpenseTask],
+    };
+
+    const combinedTrackSubmitOnboardingEmployerOrSubmitMessage: OnboardingMessage = {
+        ...onboardingEmployerOrSubmitMessage,
+        tasks: [selfGuidedTourTask, submitExpenseTask],
+    };
+
+    const onboardingPersonalSpendMessage: OnboardingMessage = {
+        message: translateLocal('onboarding.messages.personalSpendMessage'),
+
+        tasks: [selfGuidedTourTask, trackExpenseTask],
+    };
+    const combinedTrackSubmitOnboardingPersonalSpendMessage: OnboardingMessage = {
+        ...onboardingPersonalSpendMessage,
+        tasks: [selfGuidedTourTask, trackExpenseTask],
+    };
+    return {
+        ONBOARDING_MESSAGES: {
```

Comment:
```suggestion
        onboardingMessages: {
```

NAB. I think we should drop the uppercase as this is no longer defined in CONST.